src/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/StartupCode/LowLevelUTF8Encoding.cs

```diff
@@ -51,8 +52,8 @@ private unsafe static int PtrDiff(char* a, char* b)
         // kept the same as much as possible
         internal unsafe static int GetCharCount(byte* bytes, int count)
         {
-            Contract.Assert(count >= 0);
-            Contract.Assert(bytes != null);
+            Debug.Assert(count >= 0);
```

Comment:
I guess this use `Contract.Assert` because of the regular `System.Private.CoreLib\src\System\Text\UTF8Encoding.cs`  that this was derived from used it. Changing it to Assert sounds good.